packages/editor/src/store/actions.js

```diff
@@ -835,6 +835,9 @@ export const toggleDistractionFree =
 				dispatch.setIsInserterOpened( false );
 				dispatch.setIsListViewOpened( false );
 			} );
+
+			// Exit zoom out state when toggling distraction free mode.
+			unlock( registry.dispatch( blockEditorStore ) ).resetZoomLevel();
```

Comment:
Let's move this into the `registry.batch` block above.

P.S. The inline comment seems redundant here.